Source/ThirdParty/libwebrtc/CMakeLists.txt

```diff
@@ -1791,7 +1791,7 @@ if (WTF_CPU_X86_64 AND WTF_CPU_HAS_AVX2)
         Source/webrtc/modules/audio_processing/aec3/vector_math_avx2.cc
         Source/webrtc/modules/audio_processing/agc2/rnn_vad/vector_math_avx2.cc
     )
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=native -mno-avx512f")
+    set_source_files_properties(webrtc_SOURCES PROPERTIES COMPILE_OPTIONS "-mavx2;-mfma")
```

Comment:
Instead of applying `set_source_files_properties` to all the `webrtc_SOURCES ` I think it would be better to create a new list named `webrtc_avx_SOURCES` or similar and then only apply this flags to that specific files.

Then the new list `webrtc_avx_SOURCES`  can be concatenated with the `webrtc_SOURCES ` one.

So the idea is to only build with compiler flags `"-mavx2;-mfma"` only this very specific source files related to `avx2`